Core/AppRuntime/Source/AppRuntimeWin32.cpp

```diff
@@ -7,12 +7,20 @@
 
 namespace Babylon
 {
+    namespace
+    {
+        constexpr size_t FILENAME_BUFFER_SIZE = 1024;
+    }
+
     void AppRuntime::RunPlatformTier()
     {
         HRESULT hr = CoInitializeEx(nullptr, COINIT_APARTMENTTHREADED);
         assert(SUCCEEDED(hr));
         auto coInitScopeGuard = gsl::finally([] { CoUninitialize(); });
 
-        RunEnvironmentTier();
+        char filename[FILENAME_BUFFER_SIZE];
+        DWORD result = GetModuleFileNameA(nullptr, filename, FILENAME_BUFFER_SIZE);
```

Comment:
```suggestion
        DWORD result = GetModuleFileNameA(nullptr, filename, std::size(filename));
```

Perhaps don't need a constexpr for buffer size anymore.